starter-code/Bookshelf/app/src/main/java/ly/generalassemb/drewmahrt/bookshelf/MainActivity.java

```diff
@@ -52,10 +57,28 @@ public long getItemId(int position) {
             @Override
             public View getView(int position, View convertView, ViewGroup parent) {
                 //TODO: Update the view
+                if (convertView == null){
+                    LayoutInflater inflater = LayoutInflater.from(MainActivity.this);
+                    convertView = inflater.inflate(android.R.layout.simple_list_item_1, null);
+                }
+                TextView textView = (TextView) convertView.findViewById(android.R.id.text1);
+                textView.setText("Title: " + mBookList.get(position).getTitle() +
+                        "\nAuthor: " + mBookList.get(position).getAuthor());
```

Comment:
This approach works great to show title and author within the same TextView. If you wanted to format them differently, however, like make title bigger than the author, you would need two text views. Check out the solution code for an example of how to do this.